soh/src/overlays/actors/ovl_En_G_Switch/z_en_g_switch.h

```diff
@@ -56,6 +64,11 @@ typedef struct EnGSwitch {
     /* 0x0178 */ s8 objIndex;
     /* 0x017C */ ColliderCylinder collider;
     /* 0x01C8 */ EnGSwitchEffect effects[100];
+    // #region SOH [Randomizer]
+    /*        */ RandomizerCheck rc;
+    /*        */ RandomizerGet rg;
+    /*        */ GetItemEntry* giEntry;
+    /*        */ RandomizerInf randInfFlag;
```

Comment:
Could you not derive these 3 from just storing the RC?

Reply:
Well, the randomizer get is for the Silver _Trackers_, not Silver Rupees. That rando get is for identifying which room it's for. It's get a little messy since the rupees and the tracker share an actor type.

The other two I store after deducing them once because if I don't store them somewhere I have to deduce again every frame for the drawing and update functions to work properly.

Reply:
You would know better than me actually, is there a good place to store them with the hook setup without extending the vanilla struct like this?